Algorithm.CSharp/Alphas/GlobalEquityMeanReversionIBSAlpha.cs

```diff
@@ -0,0 +1,149 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Algorithm.Framework;
+using QuantConnect.Algorithm.Framework.Alphas;
+using QuantConnect.Algorithm.Framework.Execution;
+using QuantConnect.Algorithm.Framework.Portfolio;
+using QuantConnect.Algorithm.Framework.Risk;
+using QuantConnect.Algorithm.Framework.Selection;
+using QuantConnect.Data;
+using QuantConnect.Data.UniverseSelection;
+using QuantConnect.Orders.Fees;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuantConnect.Algorithm.CSharp.Alphas
+{
+    /// <summary>
+    /// Equity indices exhibit mean reversion in daily returns. The Internal Bar Strength indicator (IBS), 
+    /// which relates the closing price of a security to its daily range can be used to identify overbought
+    /// and oversold securities.
+
+    /// This alpha ranks 33 global equity ETFs on its IBS value the previous day and predicts for the following day
+    /// that the ETF with the highest IBS value will decrease in price, and the ETF with the lowest IBS value 
+    /// will increase in price.
+
+    /// Source: Kakushadze, Zura, and Juan Andr�s Serur. �4. Exchange-Traded Funds (ETFs).� 151 Trading Strategies, Palgrave Macmillan, 2018, pp. 90�91.
+
+    /// <br><br>This alpha is part of the Benchmark Alpha Series created by QuantConnect which are open sourced so the community and client funds can see an example of an alpha. 
```

Comment:
Please remove HTML but leave the text. No need to link to himself so can remove second sentence.